config/ontoemma_nn_all.json

```diff
@@ -17,89 +17,85 @@
       }
     }
   },
-  "train_data_path": "/net/nfs.corp/s2-research/scigraph/data/ontoemma/training/ontoemma.context.train",
-  "validation_data_path": "/net/nfs.corp/s2-research/scigraph/data/ontoemma/training/ontoemma.context.dev",
+  "train_data_path": "data/ontoemma.enriched.train",
+  "validation_data_path": "data/ontoemma.enriched.dev",
   "model": {
     "type": "ontoemmaNN",
-    "name_text_field_embedder": {
-            "tokens": {
-                    "type": "embedding",
-                    "pretrained_file": "/net/nfs.corp/s2-research/scigraph/data/ontoemma/embeddings/weights100.txt.gz",
-                    "embedding_dim": 100,
-                    "trainable": false
-            },
-            "token_characters": {
-                    "type": "character_encoding",
-                    "embedding": {
-                      "embedding_dim": 50
-                    },
-                    "encoder": {
-                      "type": "cnn",
-                      "embedding_dim": 50,
-                      "num_filters": 50,
-                      "ngram_filter_sizes": [4, 5]
-                    },
-                    "dropout": 0.2
-            }
+    "name_embedder": {
+      "tokens": {
+        "type": "embedding",
+        "pretrained_file": "data/embeddings/weights300.txt.gz",
```

Comment:
Should we upload all data files needed to use this project somewhere and include a script that downloads them in `data/` subdirectory? Otherwise, how would people be able to use this config file?

Reply:
Yes, totally; will add to to-do